conf/models.yml

```diff
@@ -21,3 +21,43 @@ model: &least_squares
 
 model: &xgboost
     type: xgboost
+
+model: &averaging
+    type: averaging
+
+model: &decision_tree_4
+    type: decision tree
+    max_depth: 4
+
+model: &decision_tree
+    type: decision tree
+    max_depth: None
+
+
+model: &mlp
+    type: grad mlp
+    dimensions: [20, 100, 1]
```

Comment:
This should be [100], The in dimension and out dimension are filled in by the parsing code

Reply:
It says outdated because this ended up nested in another argument, but the change is still necessary

Reply:
fixed.